### DISCUSSION TITLE
About Mnemonic Phrase

### DISCUSSION THREAD

**Discussion by 0xJchen** [phantom/docs#117] [Help & Support] [answered]
### Summary

How phantom parse the 12words Mnemonic Phrase to get the final public address? That is, I first have a 12 words phrase generated by phantom when creating a new wallet, how can I get the solana address corresponds to that seed phrase?

I tried solana-sdk, `solana-keygen recover` will ask me to provide a seed phrase, while after that, it asks me for another associated passphrase and I am confused about it. It would be great if phantom team can provide an example.

Thank you!

### Steps to Reproduce

1. download phantom and create a new wallet, receive the 12 words seed phrase
2. import the 12 words seed phrase to a new phantom wallet, assure the corresponding address is the same.

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hey @0xJchen,

Thank you for writing in!

Here is info on our derivation paths:

https://help.phantom.app/hc/en-us/articles/12988493966227-What-derivation-paths-does-Phantom-wallet-support-
